safehttp/plugins/csp/csp_test.go

```diff
@@ -0,0 +1,253 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package csp
```

Comment:
Most of the other plugins have the test in `{plugin}_test`, do you need this one to be in the `csp` package?

Reply:
By having the tests in the `csp` package I can write unit tests for the `serialize` functions of the policies in `TestSerialize`. If the `csp_test` package would be used then I would have to put all those tests in `TestBefore` which would have made them less readable.